omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/scripts/check_token_using_endpoint.lua.erb

```diff
@@ -0,0 +1,24 @@
+ngx.req.read_body()
+local res
+
+if ngx.var.request_method == "POST" then
```

Comment:
Fully agreed. I will remove the `GET` path for now. We can add that later on when we actually need it.